common-bom/pom.xml

```diff
@@ -13,16 +13,16 @@
     <name>Common BOM</name>
     <description>Common bill of materials defining standard versions for dependencies</description>
 
-    <!-- ===== NOTES ===== The dependencies defined here are NOT included in
```

Comment:
There appears to be a lot of changes in here that are just formatting. I think we really need to make sure we've got the same style template.